pilot/pkg/model/telemetry.go

```diff
@@ -410,32 +450,46 @@ func (t *Telemetries) telemetryFilters(proxy *Proxy, class networking.ListenerCl
 	return res
 }
 
-// mergeLogs returns the set of providers for the given logging configuration.
-// This currently is just the names of providers as there is no access logging configuration, but
-// in the future it will likely be extended
-func mergeLogs(logs []*tpb.AccessLogging, mesh *meshconfig.MeshConfig) (sets.Set, *tpb.AccessLogging_Filter) {
-	providers := sets.New()
+// mergeLogs returns the merged configuration for the given telemetry logging configuration.
+func mergeLogs(logs []*tpb.AccessLogging, mesh *meshconfig.MeshConfig) *LoggingSpec {
+	cfg := &LoggingSpec{
+		Client: &LoggingConfig{
+			ProviderNames: sets.New(),
+		},
+		Server: &LoggingConfig{
+			ProviderNames: sets.New(),
+		},
+	}
 
+	providerNames := mesh.GetDefaultProviders().GetAccessLogging()
```

Comment:
move after if len(logs) == 0 {

Reply:
this merged this two
![image](https://user-images.githubusercontent.com/4354057/165056244-751d6dec-60e7-4788-975a-4c93b4177f18.png)